apps/odyssey-stats/src/app.jsx

```diff
@@ -47,7 +47,7 @@ async function AppBoot() {
 	if ( ! window.location?.hash ) {
 		window.location.hash = `#!/stats/day/${ siteId }`;
 	}
-	registerStatsPages( config( 'admin_page_base' ) );
+	registerStatsPages( window.location.pathname + window.location.search );
```

Comment:
Any potential XSS risk here, as both can be easily manipulated?

Reply:
Good question! The page base is not rendered anywhere in the dom, and is only used to offset route prefix and determine whether a route should be handled by page.js, so it should fine.